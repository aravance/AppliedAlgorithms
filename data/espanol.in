352
z
y
x
w
v
u
t
s
rr
r
q
p
o
~n
n
m
ll
l
k
j
i
h
g
f
e
d
ch
c
b
a
aruba
chlmbogus
aaabogusz
aaabogusy
aaabogusx
aaabogusw
aaabogusv
aaabogusu
aaabogust
aaaboguss
aaabogusrr
aaabogusr
aaabogusq
aaabogusp
aaaboguso
aaabogus~n
aaabogusn
aaabogusm
aaabogusll
aaabogusl
aaabogusk
aaabogusj
aaabogusi
aaabogush
aaabogusg
aaabogusf
aaaboguse
aaabogusd
aaabogusch
aaabogusc
aaabogusb
aaabogusa
zbogus
ybogus
xbogus
wbogus
vbogus
ubogus
tbogus
sbogus
rrbogus
rbogus
qbogus
pbogus
obogus
~nbogus
nbogus
mbogus
llbogus
lbogus
kbogus
jbogus
ibogus
hbogus
gbogus
fbogus
ebogus
dbogus
chbogus
cbogus
bbogus
abogus
clnrbogus
chll~nrrbogus
cibogus
llbogus
lmbogus
rrbogus
rsbogus
n~nnbogus
nnnbogus
chico
cubana
ofrece
linea
gratis
contamos
monolingue
sinonimos
antonimos
formulario
abajo
o
arriba
elige
espasa
significados
version
mil
paginas
suplemento
a~nade
muchas
traducciones
todavia
no
encuentras
tu
termino
puedes
preguntar
sala
vocabulario
bilingues
portugues
dos
editorial
lengua
nueva
adquisicion
lengua
espa~nola
espasa
calpe
ademas
poder
cualquier
disfrutar
enormes
ventajas
sobre
papel
puedes
realizar
busquedas
selectivas
permiten
mayor
flexibilidad
por
ejemplo
posible
contienen
insecto
quieres
buscar
todas
palabras
relacionadas
astronomia
busca
la
abreviatura
astron
sinonimos
antonimos
adicion
estar
disponible
busquedas
normales
todos
espa~noles
este
cada
hay
lista
otras
en
incluyen
dicha
palabra
entradas
sinonimos
antonimos
frances
son
bastante
extensos
siguen
creciendo
tiene
el
italiano
ingles
tiene
casi
seguimos
aumentando
mejorando
dia
lenguaje
idiomas
diccionarios
ofrecemos
los
idiomas
donde
mas
miembros
registrados
se
ayudan
con
preguntas
respuestas
sobre
traduccion
gramatica
estilo
pronunciacion
etimologia
palabras
frases
tambien
existe
una
seccion
recursos
unos
foros
de
terminologia
especializada
como
el
foro
juridico
y
el
foro
medico
si
lo
que
estas
buscando
es
mas
que
una
traduccion
unete
a
nosotros
herramientas
la
barra
herramientas
para
usar
los
diccionarios
de
desde
suizo
numero
mundo
galardonado
asociacion
profesionales
ademas
recibir
otros
reconocimientos
quien
gano
cuatro
torneos
se
realizaron
estados
unidos
tenista
premiado
suizo
quien
permanecio
sitio
mundial
distinguido
gracias
trabajo
dentro
fundacion
ademas
merecedor
reconocimiento
primer
tenista
desde
ganar
menos
diez
titulos
durante
tres
a~nos
consecutivos
recibio
por
cuarto
votado
aficionados
pagina
oficial
su
parte
serbio
recibio
premio
del
mientras
que
aleman
fue
reconocido
tenista
estadounidense
recibio
galardon
al
a~no
ademas
sus
compatriotas
fueron
elegidos
como
mejor
pareja
dobles
donotreadthisline

