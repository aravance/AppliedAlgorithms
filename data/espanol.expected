a
a
aaabogusa
aaabogusb
aaabogusc
aaabogusch
aaabogusd
aaaboguse
aaabogusf
aaabogusg
aaabogush
aaabogusi
aaabogusj
aaabogusk
aaabogusl
aaabogusll
aaabogusm
aaabogusn
aaabogus~n
aaaboguso
aaabogusp
aaabogusq
aaabogusr
aaabogusrr
aaaboguss
aaabogust
aaabogusu
aaabogusv
aaabogusw
aaabogusx
aaabogusy
aaabogusz
abajo
abogus
abreviatura
ademas
ademas
ademas
ademas
adicion
adquisicion
aficionados
al
aleman
antonimos
antonimos
antonimos
a~nade
a~no
a~nos
aruba
arriba
asociacion
astron
astronomia
aumentando
ayudan
b
barra
bastante
bbogus
bilingues
busca
buscando
buscar
busquedas
busquedas
c
cada
calpe
casi
cbogus
cibogus
clnrbogus
como
como
compatriotas
con
consecutivos
contamos
contienen
creciendo
cualquier
cuarto
cuatro
cubana
ch
chbogus
chico
chlmbogus
chll~nrrbogus
d
dbogus
de
de
del
dentro
desde
desde
dia
diccionarios
diccionarios
dicha
diez
disfrutar
disponible
distinguido
dobles
donde
dos
durante
e
ebogus
editorial
ejemplo
el
el
el
elegidos
elige
en
encuentras
enormes
entradas
es
espa~nola
espa~noles
espasa
espasa
especializada
estados
estadounidense
estar
estas
este
estilo
etimologia
existe
extensos
f
fbogus
flexibilidad
formulario
foro
foro
foros
frances
frases
fue
fueron
fundacion
g
galardon
galardonado
ganar
gano
gbogus
gracias
gramatica
gratis
h
hay
hbogus
herramientas
herramientas
i
ibogus
idiomas
idiomas
incluyen
ingles
insecto
italiano
j
jbogus
juridico
k
kbogus
l
la
la
lbogus
lengua
lengua
lenguaje
linea
lista
lmbogus
lo
los
los
ll
llbogus
llbogus
m
mas
mas
mayor
mbogus
medico
mejor
mejorando
menos
merecedor
miembros
mientras
mil
monolingue
muchas
mundial
mundo
n
nbogus
nnnbogus
n~nnbogus
no
normales
nosotros
nueva
numero
~n
~nbogus
o
o
obogus
oficial
ofrece
ofrecemos
otras
otros
p
pagina
paginas
palabra
palabras
palabras
papel
para
pareja
parte
pbogus
permanecio
permiten
poder
por
por
portugues
posible
preguntar
preguntas
premiado
premio
primer
profesionales
pronunciacion
puedes
puedes
q
qbogus
que
que
que
quien
quien
quieres
r
rbogus
realizar
realizaron
recibio
recibio
recibio
recibir
reconocido
reconocimiento
reconocimientos
recursos
registrados
relacionadas
respuestas
rsbogus
rr
rrbogus
rrbogus
s
sala
sbogus
se
se
seccion
seguimos
selectivas
serbio
si
significados
siguen
sinonimos
sinonimos
sinonimos
sitio
sobre
sobre
son
su
suizo
suizo
suplemento
sus
t
tambien
tbogus
tenista
tenista
tenista
termino
terminologia
tiene
tiene
titulos
todas
todavia
todos
torneos
trabajo
traduccion
traduccion
traducciones
tres
tu
u
ubogus
una
una
unete
unidos
unos
usar
v
vbogus
ventajas
version
vocabulario
votado
w
wbogus
x
xbogus
y
y
ybogus
z
zbogus
